lib/gems/pending/util/postgres_admin.rb

```diff
@@ -245,24 +245,29 @@ def self.vacuum(opts)
     args[:full]    = nil if opts[:full]
     args[:verbose] = nil if opts[:verbose]
     args[:table]   = opts[:table] if opts[:table]
-    runcmd("vacuumdb", opts, args)
+    run_command("vacuumdb", opts, args)
   end
 
   def self.reindex(opts)
     args = {}
     args[:table] = opts[:table] if opts[:table]
-    runcmd("reindexdb", opts, args)
+    run_command("reindexdb", opts, args)
   end
 
-  def self.runcmd(cmd_str, opts, args)
-    runcmd_with_logging(cmd_str, opts, combine_command_args(opts, args))
+  def self.run_command(cmd_str, opts, args)
+    run_command_with_logging(cmd_str, opts, combine_command_args(opts, args))
   end
 
-  def self.runcmd_with_logging(cmd_str, opts, params = {})
+  def self.run_command_with_logging(cmd_str, opts, params = {})
```

Comment:
Similarly here I renamed it for clarity.  There are exactly 2 callers in core specs that stub runcmd_with_logging, which is why i need the alias below, but I'll remove that in a follow up.